### ISSUE TITLE
Browser search all open databases for matching TOTP entry

### ISSUE THREAD

**Issue by xMorioh** [keepassxreboot/keepassxc#10293]
## Summary
I'm using KeePassXC and have two Databases, one for Username and Password credentials and one Database for all TOTPs for those Websites,

I'd like to use the feature "Search in all opened databases for matching credentials" in combination with the Browser Plugin but i sadly cannot really do it.

## Examples
When i try to login to a website then there is a good chance that either the Username and Passwort will not fill in or the TOTP won't instead.
It seems that there is no check in KeePassXC for two entries that exist in separate databases to return a requested value if it is only present in one of those entries. So for example KeePassXC will not start checking if the requested TOTP is present in another database if it checked the first entry it found where no TOTP was present and vise versa for Username and Password.

## What i've tried and how i set it up
- I tried it with refreshing both database root group IDs.
- I am using the exact same Link in both entries.
- Both Databases are unlocked when this happens.
- Deactivating "Search in all opened databases for matching credentials" and me switching databases manually will always work perfectly fine.
- All Fields on a given website get detected without any issues, KeepassXC knows exactly when it requires a Username value and when it requires a Password or TOTP so that is not the issue here!

## Suggestion
I'd like to have an option or make it the default that if i click on the TOTP button for example in a browser field that it will check all entries in all Databases as the Toggle in "Search in all opened databases for matching credentials" suggests it would do.
Same goes for Username and Password.
So basically this would require a "did not find requested value on first entry, if present then checking next entry for current URL" in the code.

**Comment by droidmonkey** [comment]
There are very good reasons for the behavior you witness, we obviously want consistency through a multi step login process so randomly switching between two entries is definitely not desirable. I think it is reasonable, though, that KeePassXC at least attempt a new search for a match entry if TOTP is required and the current chosen entry doesn't have TOTP configured.

**Comment by Kyren223** [comment]
Any updates on this? new user to KeePassXC and really missing this, would be awesome to see this added.

**Comment by varjolintu** [comment]
> Any updates on this? new user to KeePassXC and really missing this, would be awesome to see this added.

Not yet.
